### ISSUE TITLE
SEM plots: Include "1" in triangle if means in parameters

### ISSUE THREAD

**Issue by bwiernik** [easystats/see#113]
Plots for models with a mean structure typically include a `1` in a triangle with the paths indicating the means/intercepts. Could this be added if the parameters() output includes means?

**Comment by DominiqueMakowski** [comment]
> Could this be added if the parameters() output includes means?

Yes with some additional code, I don't see why not. But it would require `model_parameters()` to return them first

**Comment by strengejacke** [comment]
Do you have a reprex, so I know what should be returned?

**Comment by strengejacke** [comment]
According to the docs, you can return the mean component already: https://easystats.github.io/parameters/reference/model_parameters.lavaan.html

**Comment by bwiernik** [comment]
Yes you can already get means into parameters by adding "mean" to component. This is currently ignored by the plotting function
